File: M2_exercicios/A45-4/SalaReunioes/SalaReunioes.Domain/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SalaReunioes.Domain
{
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Job { get; set; }
        public string Phone { get; set; }

        public Employee(string name, string job, string phone)
        {
            Name = name;
            Job = job;
            Phone = phone;
        }

        public Employee()
        {

        }

        public override string ToString()
        {
            return $"{Id} - {Name} - {Job} - {Phone}";
        }
    }
}

File: M3_exercicios/A19/ex03/classlib/Employee.ts
class Employee {
    protected _name: string;
    protected _salary: number;

    constructor(name: string, salary: number) {
        this._name = name;
        this._salary = salary;
    }

    public get name(): string {
        return this._name;
    }

    public set name(name: string) {
        this._name = name;
    }

    public get salary(): number {
        return this._salary;
    }

    public set salary(salary: number) {
        this._salary = salary;
    }
}